src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/mapper/intervener/IntervenerFourToIntervenerDetailsMapper.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.mapper.intervener;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.wrapper.IntervenerFourWrapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.intervener.IntervenerDetails;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor
+public class IntervenerFourToIntervenerDetailsMapper {
+
+    public IntervenerDetails mapToIntervenerDetails(IntervenerFourWrapper intervenerFourWrapper) {
+        return IntervenerDetails.builder()
+            .intervenerName(intervenerFourWrapper.getIntervener4Name())
+            .intervenerAddress(intervenerFourWrapper.getIntervener4Address())
+            .intervenerEmail(intervenerFourWrapper.getIntervener4Email())
+            .intervenerPhone(intervenerFourWrapper.getIntervener4Phone())
+            .intervenerRepresented(intervenerFourWrapper.getIntervener4Represented())
+            .intervenerDateAdded(intervenerFourWrapper.getIntervener4DateAdded())
+            .intervenerSolName(intervenerFourWrapper.getIntervener4SolName())
```

Comment:
new add field by is not there? firm and reference

Reply:
I've updated this